src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -887,8 +887,8 @@ protected void processCopyLines(List<String> dockerfileLines, String buildContex
                 // Break the Dockerfile line down into segments based on any amount of whitespace.
                 // The command must be to the left of any comments.
                 String[] cmdSegments = trimLine.split("#")[0].split("\\s+");
-                // If the line starts with COPY
-                if (cmdSegments[0].equalsIgnoreCase("COPY")) {
+                // If the line starts with COPY or ADD
+                if (cmdSegments[0].equalsIgnoreCase("COPY") || cmdSegments[0].equalsIgnoreCase("ADD")) {
```

Comment:
Update the exceptions and warning messages in this method.

Reply:
ok